prototypes/anim_polygons_data/shared/png2svg.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import argparse
+import xml.etree.ElementTree as ET
+from PIL import Image, ImageOps
+
+PASSEPARTOUT = 10
+TMP_DIR = ""
+TMP_SVG_FILE = "vtrace_tmp.svg"
+
+xmlns = {"svg": "http://www.w3.org/2000/svg"}
+
+
+def run_vtrace(img):
+    return os.system("vtracer --colormode bw -c 30"
+                     " --color_precision 8 --filter_speckle 15"
+                     f" --gradient_step 0 --mode polygon --input {img}"
+                     f" --output {os.path.join(TMP_DIR, TMP_SVG_FILE)}")
+
+
+def make_passpepartout(img):
+    img_name = os.path.join(TMP_DIR, "passepartout.png")
+    ImageOps.expand(ImageOps.grayscale(Image.open(img)),
+                    PASSEPARTOUT, 0).save(img_name, "png")
+    return img_name
+
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser(description="Process rendered animation "
```

Comment:
You can break line just after left parenthesis to gain more space to write the description.